ethtest/mock_data.go

```diff
@@ -42,6 +42,7 @@ func CreateTestTransaction(t *testing.T) txcontext.TxContext {
 	}
 	to := common.HexToAddress("0x10")
 	return &stateTestContext{
+		logHash: common.HexToHash("0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347"), // empty logs rlp hash
```

Comment:
nit: To make points like this clear, I like to write stuff like this:
```suggestion
		logHash: rlpHash([]*types.Log{})
```
Contains the same information and if -- for whatever reason -- the implementation of the RLP hash or the log type changes, it is automatically fixed.